data/json/npcs/BGSS_talk_common.json

```diff
@@ -0,0 +1,965 @@
+        "condition": { "npc_has_trait": "BGSS_Rural_1" }
+      },
+      {
+        "text": "<BGSS_intro_question>",
+        "topic": "BGSS_RURAL_2_STORY1",
+        "condition": { "npc_has_trait": "BGSS_Rural_2" }
+      }
+    ]
+  },
+  {
+    "type": "effect_type",
+    "id": "player_BGSS_SAIDNO",
+    "//": "Defined here in the conversation section because this should be the only time this effect is referenced: it's specifically for survival stories only.",
+    "name": [ "Story Denied" ],
+    "desc": [
+      "AI tag used when you already asked an NPC for a detail in their survival story and they don't want to tell you again right now.  This is a bug if you have it."
+    ]
+  },
+  {
+    "id": "BGSS_CONFUSED_1_STORY1",
+    "type": "talk_topic",
+    "dynamic_line": "I don't even <swear> know anymore.  I have no <swear> idea what is going on.  I'm just doing what I can to stay alive.  The world ended and I bungled along not dying, until I met you.",
+    "responses": [ { "text": "Huh.", "topic": "TALK_FRIEND" } ]
+  },
+  {
+    "id": "BGSS_NO_PAST_1_STORY1",
+    "type": "talk_topic",
+    "dynamic_line": "Before <the_cataclysm>? Who cares about that? This is a new world, and yeah, it's pretty <shitty>.  It's the one we've got though, so let's not dwell in the past when we should be making the best of what little we have left.",
+    "responses": [ { "text": "I can respect that.", "topic": "TALK_FRIEND" } ]
+  },
+  {
+    "id": "BGSS_NO_PAST_2_STORY1",
+    "type": "talk_topic",
+    "dynamic_line": "To be honest...  I don't really remember.  I remember vague details of my life before the world was like this, but <the_cataclysm> itself? It's all a blur.  I don't know how I got where I am now, or how any of this happened.  I think something pretty bad must have happened to me.  Or maybe I was just hit in the head really hard.  Or both.  Both seems likely.",
+    "responses": [ { "text": "Huh.", "topic": "TALK_FRIEND" } ]
+  },
+  {
+    "id": "BGSS_RELIGIOUS_1_STORY1",
+    "type": "talk_topic",
+    "dynamic_line": "My story.  Huh.  It's nothing special.  I had people, but they've risen to be with the Lord.  I don't understand why He didn't take me too, but I suppose it'll all be clear in time.",
+    "responses": [
+      { "text": "Do you mean in a religious sense, or...?", "topic": "BGSS_RELIGIOUS_1_STORY2" },
+      { "text": "<done_conversation_section>", "topic": "TALK_FRIEND" },
+      { "text": "<end_talking>", "topic": "TALK_DONE" }
+    ]
+  },
+  {
+    "id": "BGSS_RELIGIOUS_1_STORY2",
+    "type": "talk_topic",
+    "dynamic_line": "Of course.  It's clear enough, isn't it? That...  that end, was the Rapture.  I'm still here, and I still don't understand why, but I will keep Jesus in my heart through the Tribulations to come.  When they're past, I'm sure He will welcome me into the Kingdom of Heaven.  Or...  or something along those lines.  It's not going exactly like I thought it would, but that's prophecy for you.",
```

Comment:
```suggestion
    "dynamic_line": "Of course.  It's clear enough, isn't it?  That...  that end, was the Rapture.  I'm still here, and I still don't understand why, but I will keep Jesus in my heart through the Tribulations to come.  When they're past, I'm sure He will welcome me into the Kingdom of Heaven.  Or...  or something along those lines.  It's not going exactly like I thought it would, but that's prophecy for you.",
```